products/shopware.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Shopware
+category: server-app
+tags: php-runtime
+iconSlug: shopware
+permalink: /shopware
+versionCommand: ./bin/console -V
+releasePolicyLink: https://www.shopware.com/en/news/shopware-6-versioning-strategy/
+changelogTemplate: "https://www.shopware.com/en/changelog/#release-{{'__LATEST__' | replace:'.','-'}}"
+releaseLabel: "{{'__RELEASE_CYCLE__' | replace:'.','-'}}"
+releaseColumn: true
+releaseDateColumn: true
+releases:
+-   releaseCycle: "6.5"
+    releaseDate: 2023-05-03
+    eol: false
+    latest: "6.5.5.1"
+    latestReleaseDate: 2023-09-05
+
+-   releaseCycle: "6.4"
+    eol: 2021-05-04
```

Comment:
This eol is probably not correct, since the latest release, version 6.4.20.1, was released April 17th, 2023. https://www.shopware.com/en/changelog/#release-6-4-20-1